gfx.h

```diff
@@ -1770,7 +1773,7 @@ class GfxInternal
             resource_desc.Flags = D3D12_RESOURCE_FLAG_ALLOW_UNORDERED_ACCESS;
             break;
         }
-        if(createResource(allocation_desc, resource_desc, resource_state, &gfx_buffer.allocation_, IID_PPV_ARGS(&gfx_buffer.resource_)) != kGfxResult_NoError)
+        if(createResource(allocation_desc, resource_desc, resource_state, nullptr, &gfx_buffer.allocation_, IID_PPV_ARGS(&gfx_buffer.resource_)) != kGfxResult_NoError)
```

Comment:
Why the additional `nullptr` here?

Reply:
I added an additional parameter to the `createResource` function as that internally sets the clear value on textures so it needed the requested clear value to be passed in. Since that function is also used to create generic buffers then in that case just pass nullptr